index.html

```diff
@@ -0,0 +1,222 @@
+    <p>You may also compound the statements using else if to have multiple conditions tested in sequence, as follows:</p>
+    <code>if (condition_1) { statement_1; } else if (condition_2) {
+      statement_2; } else if (condition_n) { statement_n; } else {
+      statement_last; }</code>
+      <p>In the case of multiple conditions only the first logical condition which evaluates to true will be executed. To execute multiple statements, group them within a block statement ({ ... }) . In general, it's good practice to always use block statements, especially when nesting if statements:</p>
+      <code>if (condition) { statement_1_runs_if_condition_is_true;
+        statement_2_runs_if_condition_is_true; } else {
+          statement_3_runs_if_condition_is_false;
+        statement_4_runs_if_condition_is_false; }</code>
+        <p>It is advisable to not use simple assignments in a conditional expression, because the assignment can be confused with equality when glancing over the code. For example, do not use the following code:</p>
+        <code>if (x = y) { /* statements here */ }</code>
+      <p>If you need to use an assignment in a conditional expression, a common practice is to put additional parentheses around the assignment. For example:</p>
+      <code>if ((x = y)) { /* statements here */ }</code>
+    </section>
+
+    <section class="main-section" id="while_statement">
+      <header>while statement</header>
+      <p>A while statement executes its statements as long as a specified condition evaluates to true. A while statement looks as follows:</p>
+      <code>while (condition) statement</code>
+        <p>If the condition becomes false, statement within the loop stops executing and control passes to the statement following the loop.</p>
+        <p>The condition test occurs before statement in the loop is executed. If the condition returns true, statement is executed and the condition is tested again. If the condition returns false, execution stops and control is passed to the statement following while.</p>
+        <p>To execute multiple statements, use a block statement ({ ... }) to group those statements.</p>
+        <p>Example:</p>
+        <p>The following while loop iterates as long as n is less than three:</p>
+      <code>var n = 0; var x = 0; while (n < 3) { n++; x += n; }</code>
+      <p>With each iteration, the loop increments n and adds that value to x. Therefore, x and n take on the following values:</p>
+      <ul>
+        <li>After the first pass: n = 1 and x = 1</li>
+        <li>After the second pass: n = 2 and x = 3</li>
+        <li>After the third pass: n = 3 and x = 6</li>
+      </ul>
+      <p>After completing the third pass, the condition n < 3 is no longer true, so the loop terminates.</p>
+    </section>
+
+    <section class="main-section" id="Function_declarations">
+    <header>Function declarations</header>
```

Comment:
Replace your `header` element with heading element(h1, h2, h3, h4, h5, h6)

Reply:
- [ ] Kindly always include an explanation, or a "why", for each requested change, this will make your comments more useful and clear. eg why using the header as a substitute for your ```<hx>``` is not a good thing